app/code/core/Mage/Core/Model/Resource/Config.php

```diff
@@ -101,7 +101,7 @@ public function loadToXml(Mage_Core_Model_Config $xmlConfig, $condition = null)
             if ($r['scope'] !== 'default') {
                 continue;
             }
-            $value = str_replace($substFrom, $substTo, $r['value']);
+            $value = empty($r['value']) ? $r['value'] : str_replace($substFrom, $substTo, $r['value']);
```

Comment:
This added empty-check must also be applied in line 121 and 150 for website and store scope, respectively.